apps/judicial-system/web/src/components/AppealAlertBanner/AppealAlertBanner.tsx

```diff
@@ -28,79 +27,61 @@ interface Props {
   workingCase: TempCase
 }
 
-interface AppealInfo {
-  canBeAppealed: boolean
-  hasBeenAppealed: boolean
-  appealDeadline?: string
-  appealedByRole?: UserRole
-  appealedDate?: string
-}
-
-export const getAppealInfo = (workingCase: TempCase): AppealInfo => {
-  const {
-    courtEndTime,
-    appealState,
-    isAppealDeadlineExpired,
-    accusedAppealDecision,
-    prosecutorAppealDecision,
-    prosecutorPostponedAppealDate,
-    accusedPostponedAppealDate,
-  } = workingCase
-
-  const canBeAppealed = Boolean(
-    courtEndTime &&
-      !appealState &&
-      !isAppealDeadlineExpired &&
-      (accusedAppealDecision === CaseAppealDecision.POSTPONE ||
-        prosecutorAppealDecision === CaseAppealDecision.POSTPONE),
-  )
-
-  const hasBeenAppealed = Boolean(
-    appealState && appealState === CaseAppealState.Appealed,
-  )
-
-  const appealedByRole = prosecutorPostponedAppealDate
-    ? UserRole.Prosecutor
-    : accusedPostponedAppealDate
-    ? UserRole.Defender
-    : undefined
-
-  const appealedDate =
-    appealedByRole === UserRole.Prosecutor
-      ? prosecutorPostponedAppealDate ?? undefined
-      : accusedPostponedAppealDate ?? undefined
-
-  const appealDeadline = courtEndTime
-    ? getAppealEndDate(courtEndTime ?? '')
-    : undefined
-
-  return {
-    hasBeenAppealed,
-    canBeAppealed,
-    appealedByRole,
-    appealedDate,
-    appealDeadline,
-  } as AppealInfo
-}
-
 const AppealAlertBanner: React.FC<Props> = (props) => {
   const { formatMessage } = useIntl()
   const { user } = useContext(UserContext)
 
-  const limitedAccess = router.pathname.includes(DEFENDER_ROUTE)
-
   const { workingCase } = props
+
   const {
+    prosecutorStatementDate,
+    defenderStatementDate,
+    statementDeadline,
+    hasBeenAppealed,
     appealedByRole,
     appealedDate,
     canBeAppealed,
-    hasBeenAppealed,
     appealDeadline,
-  } = getAppealInfo(workingCase)
+  } = workingCase
 
   let alertTitle, alertLinkTitle, alertLinkHref, alertDescription
 
-  if (hasBeenAppealed) {
+  const isCourtRoleUser = isCourtRole(user?.role)
```

Comment:
I was a little unsure about this, should this maybe only be the judge and not a court role user?

Reply:
I think this is correct. I'll accept it and you can ask Gudny about it and change it if it's not.

Reply:
Confirmed 😊